src/main/java/tr/edu/ogu/ceng/InternshipManagementSystem.java

```diff
@@ -10,4 +12,8 @@ public static void main(String[] args) {
 		SpringApplication.run(InternshipManagementSystem.class, args);
 	} 
 	
+	@Bean // uygulama oluşur oluşmaz bir modelmapperManager için bir modelmapper nesnesi  oluşturulcak.
```

Comment:
yapılacaklar için TODO ya da FIXME kullanırız ama neden şimdi oluşturulmadı da sonraya bırakıldı böyle bir sınıfın oluşturulması

Reply:
Eğer burada bean oluşturmayı düşünüyorsak genel yaklaşımımız IoC olduğundan bean tanımlarını bu şekilde yapmıyoruz.